src/components/file-dropzone/examples/file-dropzone-type-filtering.scss

```diff
@@ -0,0 +1,18 @@
+.picture-dropzone {
+    min-width: 3rem;
+    min-height: 10rem;
+    background-color: rgb(var(--contrast-600));
+    border-radius: 0.5rem;
+    padding: 0.25rem;
+    margin-bottom: 0.5rem;
+}
+
+.video-picture-dropzone {
+    min-width: 3rem;
+    min-height: 10rem;
+    background-color: rgb(var(--contrast-600));
+    border-radius: 0.5rem;
+    border: 1px solid rgb(var(--contrast-800));
+    padding: 0.25rem;
+    margin-bottom: 0.5rem;
+}
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) 😊

Reply:
haha, right, did it in a hurry